### PR TITLE
Read your page on .alias.sh

### PR DESCRIPTION
**Pull request by ispeedyg**
I am new to git and have forgotten most of my dos commands. Let me try and explain what the plan was. I created the .aliases.sh in my home dir and it is the same dir as .bash_profile. in .bash_profile I placed the source dir like you said. and have your command for myip in aliases. which i dont know why I copied and pasted a new folder never happens. What i wanted to do as I am trying to learn swift and every now and then it gets a bit buggy is clean it out. so I planned to a couple of lines down place this code. with a trigger of cc. one of my problems is when i open up terminal it immediately executes the source folder. 

so i deleted the stuff I was trying to add and went for something simpler. 

I copied and pasted your #Hosts and trigger for that. here is what that looks like

Last login: Mon Oct 12 04:39:07 on ttys001
-bash: alias: nano: not found
-bash: alias: /private/etc/hosts”: not found
Speedy:~ Speedy$ hosts
-bash: “sudo: command not found

basically it executes when i open terminal. Here is a screen shot of first thing when I open terminal.

screen shot 2015-10-12 at 5 01 52 am

the plan is to add a quickKey to clear xcode with this code.

screen shot 2015-10-12 at 5 04 16 am

Now this is the first time I have pulled a request so I am not sure when I will be back on. And don't know if git sends me a message or something. this is what I was trying to do. And I get this error -> Speedy:~ Speedy$ cc
clang: error: no input files
screen shot 2015-10-12 at 5 09 39 am

Any help is appreciated.

Speedy

### CODE REVIEW

**Comment by mathiasbynens** [comment]
Please don’t use this issue tracker for support requests. Use something like https://stackoverflow.com/ instead. Thanks!

**Comment by ispeedyg** [comment]
That was my first time using the pull thing. Didn’t really know how to use it. Thanks for the help.
